### PR TITLE
Features/activejob setup #146963669

### PR DESCRIPTION
**Pull request by kaptron**

### CODE REVIEW

File: spec/jobs/short_form_attachment_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
```
Comment (chrisdolendo) [inline spec/jobs/short_form_attachment_job_spec.rb]: rad

**Comment by chrisdolendo** [comment]
right on 👍

**Comment by kaptron** [comment]
@creativecoder you're exactly right, good catch -- I just fixed `schema.rb`. It was because of me jumping between branches and my database being in different states, the schema ended up getting combined/confused.

File: app/services/cache_service.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# service class for pre-caching salesforce listings
+class CacheService
```
Comment (kaptron) [inline app/services/cache_service.rb]: @creativecoder would be interested to discuss `CacheService` with you and get your thoughts. As I mentioned last week we had to make some decisions/compromises as how to best pre-cache listings on a regular basis, and how frequently to do it. I think it's in a "good enough to deliver" place but I think it's worth one more conversation to finalize.

**Pull request merged**